python/ray/util/actor_pool.py

```diff
@@ -107,7 +110,9 @@ def get_generator():
 
         return get_generator()
 
-    def map_unordered(self, fn: Callable[[Any], Any], values: List[Any]):
+    def map_unordered(
```

Comment:
Nice. Could you fix `def map()` as well?

Reply:
👍 good idea, now added.